src/Cli/ArtifactAssembler/ArtifactAssembler.cs

```diff
@@ -365,6 +365,7 @@ private async Task CreateCliCoreToolsAsync()
 
             Console.WriteLine("Finished assembling CLI Core Tools artifacts");
             Console.WriteLine();
+            await Task.CompletedTask;
```

Comment:
The correct answer here is to remove the async key word and rename the method to `CreateCliCoreTools`. If we are not making any async calls, this should not be an async method. This comment applies to several of the changes here.